app/views/links/_links_list.html.erb

```diff
@@ -29,6 +20,24 @@
         <%= link_to tag.name, links_path(tag: tag.name), class: "btn btn-sm btn-theme" %>
       <% end %>
     </div>
+    <div class="btn-group" style="float: right; margin-top: -30px" id="<%="btn_group#{link.id}"%>">
+      <button data-toggle="dropdown" class="btn dropdown-toggle toggle_btn fa fa-list " title='Options' type="button">
+        <span class="sr-only">Toggle Dropdown</span>
+      </button>
+      <ul role="menu" class="dropdown-menu">
+        <li><%= link_to 'Edit', edit_link_path(link) %> </li>
+        <li><%= link_to 'Increase Learn Count', learn_time_path(link.id), remote: true, data: { confirm: 'Are you sure? Do you want to increase learn count?'} %></li>
+        <li><%= link_to 'Delete',link_path(link), method: 'delete',data: { confirm: 'Are you sure? Do you want to delete?'} %></li>
+      </ul>
+    </div>
+
   </div><!-- /showback -->
 <% end %>
-</div>
\ No newline at end of file
+</div>
+
+<script>
+  $('.btn-group').click(function(){
+       $('#'+this.id).addClass('open');
```

Comment:
Wouldn't this keep on adding same class each time you click or just once if class is same?